app/components/works/work_type_modal_component.html.erb

```diff
@@ -1,50 +1,64 @@
 <script>
-  document.subtypes = <%= WorkType.to_json.html_safe %>
+  document.subtypes = <%= WorkType.to_json.html_safe %>;
+  document.moreTypes = <%= WorkType.more_types.to_json.html_safe %>
 </script>
 
-<div class="modal-header">
-  <h5 class="modal-title" id="content-type-prompt">What type of content will you deposit?</h5>
-  <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-</div>
-<div class="modal-body">
-  <form method="get" action="/collections/1/work/new" data-work-type-target="form">
-    <div class="row">
-      <% types.each do |type| %>
-        <div class="work-type-box">
-          <%= render CheckboxButtonComponent.new(name: 'work_type', value: type.id, data: { action: "work-type#change" }) do %>
-            <span class="fas fa-2x fa-<%= type.icon %>"></span> <div class="work-type-button-text"><%= type.label %></div>
-          <% end %>
-        </div>
-      <% end %>
-    </div>
-
-    <template data-work-type-target="template">
-      <div class="col-md-4">
-        <div class="form-check">
-          <input type="checkbox" name="subtype[]" value="SUBTYPE_LABEL" id="subtype_SUBTYPE_ID" class="form-check-input">
-          <label for="subtype_SUBTYPE_ID" class="form-check-label">SUBTYPE_LABEL</label>
-        </div>
+<div class="modal fade" id="workTypeModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-dialog-scrollable modal-lg">
```

Comment:
I pull these surrounding divs into the component ERB, partly because it didn't seem important or useful to have to replicate these exact same lines everywhere we instantiate this component and because I needed to add the modal dialog scrollable class to this div.